docs/controls/src/imageviewer-mobile/main.qml

```diff
@@ -0,0 +1,117 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.4
+import QtQuick.Dialogs 1.2
+import QtQuick.Layouts 1.0
+import QtQuick.Controls.Material 2.1
+
+ApplicationWindow {
+    
+    // ...
+    
+    id: window
+
+    visible: true
+    width: 360
+    height: 520
+    title: qsTr("Image Viewer")
+
+    Drawer {
+        id: drawer
+
+        width: Math.min(window.width, window.height) / 3 * 2
+        height: window.height
+
+        ListView {
+            focus: true
+            currentIndex: -1
+            anchors.fill: parent
+
+            delegate: ItemDelegate {
+                width: parent.width
+                text: model.text
+                highlighted: ListView.isCurrentItem
+                onClicked: {
+                    drawer.close()
+                    model.triggered()
+                }
+            }
+
+            model: ListModel {
+                ListElement {
+                    text: qsTr("Open...")
+                    triggered: function(){ fileOpenDialog.open(); }
```

Comment:
This can just be:

`triggered:` fileOpenDialog.open()

Reply:
When 5.11 (which I'm testing against) this does not work. I get a "ListElement: cannot use script for property value". Does this behaviour change for 5.12?

Reply:
Ah, I didn't realise that this was in a ListElement. It seems it's intended to only work when it's explicitly specified:

https://codereview.qt-project.org/#/c/206314/